src/pactflow/client/tools.ts

```diff
@@ -82,5 +82,19 @@ export const TOOLS: PactflowToolParams[] = [
     zodSchema: MatrixSchema,
     handler: "getMatrix",
     clients: ["pactflow", "pact_broker"]
+  },
+  {
+    title: "PactFlow AI Credits",
```

Comment:
Is this better renamed to something like "PactFlow AI Status" for consistency with our CLI (e.g. it uses `pactflow-ai status`).